cpp/open3d/visualization/rendering/filament/FilamentCamera.h

```diff
@@ -26,14 +26,15 @@
 
 #pragma once
 
+#include <utils/Entity.h>
+
 #include "open3d/visualization/rendering/Camera.h"
 
 /// @cond
 namespace filament {
 class Camera;
 class Engine;
 }  // namespace filament
-/// @endcond
```

Comment:
Missing `@endcond`? This will cause doxygen to ignore the rest of this file.

Reply:
Oops, that was a mistake when cleaning up my first attempt. Fixed.